.travis.yml

```diff
@@ -3,13 +3,25 @@ branches:
     - master
```

Comment:
Won't this make `sqf_validator.py` only ran on master as well? Or is that branch whitelist/blacklist different from PRs?